lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -58,6 +62,30 @@ module Naming
       #   def is_even?(value)
       #   end
       #
+      # @example UseSorbetSigs: false (default)
+      #  # bad
+      #  sig { returns(String) }
+      #  def is_this_thing_on
+      #    "yes"
+      #  end
+      #
+      #  # good
```

Comment:
I would add a comment here to explain why this is good, such as:
```suggestion
      #  # good - sorbet signature is not evaluated
```